.github/workflows/pull_request.yml

```
@@ -13,11 +13,21 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: build-${{ github.ref }}
+  group: Pull-Request-${{ github.ref }}
```

If we rename it, then we can name this "Static-Analysis"

```suggestion
  group: Static-Analysis-${{ github.ref }}
```